src/Uno.UWP/ApplicationModel/Package.Other.cs

```diff
@@ -25,16 +26,18 @@ public partial class Package
 
 		private DateTimeOffset GetInstallDate() => DateTimeOffset.Now;
 
-		private string GetInstalledLocation()
+		private string GetInstalledPath()
 		{
-			if (!string.IsNullOrEmpty(AppContext.BaseDirectory))
+			if (_entryAssembly?.Location is { Length: > 0 } location)
 			{
-				return global::System.IO.Path.GetDirectoryName(AppContext.BaseDirectory) ?? "";
+				return global::System.IO.Path.GetDirectoryName(location) ?? "";
 			}
-			else
+			else if (AppContext.BaseDirectory is { Length: > 0 } baseDirectory)
```

Comment:
The reason is that in VS hosted context (e.g. a VS plugin using Uno Islands in WPF), `BaseDirectory` returns path to the hosting process, which is Visual Studio, instead of the path where the plugin is actually installed on the hard drive.